### DISCUSSION TITLE
Extract metrics from traces

### DISCUSSION THREAD

**Discussion by sa-ap** [open-telemetry/community#1072] [Community Discussions]
open-telemetry traces is integrated in our service. Goal is create custom alerts in dyntarace. Could one define new metrics from a trace in dynatrace? From "ConfigurationData.find", we would extract  "timing" section to trigger defined alerts.  Is that possible?

**Comment by jkowall** [comment]
You should ask this of your commercial vendor. Otel supports this use case with the processor we built here: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor
